translations/translated/updateqm.pri

```diff
@@ -41,7 +41,8 @@ for(file, TS_FILES_NOEXT) {
         system("LANG=C $$QMAKE_LRELEASE $${file}.ts -compress -qm $${file}.qm >> lrelease_output.txt")
     }
 }
-STATS_GENERATOR = $$shell_path("$${PWD}/generate-translation-stats.lua")
+STATS_GENERATOR = $$system_path("$${PWD}/generate-translation-stats.lua")
+message("Will use \"$${STATS_GENERATOR}\" to parse translation statistics...")
```

Comment:
Could you take this out now that it's working? It's better to have less messages, not more messages that you adapt to ignore.

Reply:
Would you accept them being just commented out? So an interested party (which, given the weird places {cygwin/FreeBSD/MSYS2} I occasionally come up with to build Mudlet will probably be me anyhow) can quickly re-enable to check - rather than having to remember what I used?

Reply:
Yeah, I think that sounds good!

_maybe we could do the same with a few others, too_...

Reply:
We shouldn't forget that it is run for all builds not just the CI ones.

Reply:
Yeah, it's the all builds that it's a problem, it has a lot of unnecessary information 99% of the time that you ignore. Like unix tools, one only needs to see the errors, not the successful operation.